src/mlagility/cli/report.py

```diff
@@ -1,521 +1,204 @@
 import os
 import csv
-import dataclasses
-from typing import List
-import yaml
+import json
+from datetime import datetime
+from pathlib import Path
+from dataclasses import dataclass
 import groqflow.common.printing as printing
 import groqflow.common.build as build
 import groqflow.common.cache as cache
-from groqflow import groqmodel
+from groqflow.groqmodel import groqmodel
+from mlagility.common import labels
 
 
-def _successCleanup(item):
-    return item if item is not None else False
+def _update_numeric_attribute(new_val, current_val, default="-"):
+    """
+    Updates a numeric attribute if needed
+    """
+    if current_val == default:
+        return new_val if new_val is not None else default
+    else:
+        return current_val
 
 
-def _numericCleanup(item):
-    return item if item is not None else "-"
+def get_estimated_groq_latency(model_folder, cache_folder):
+    """
+    Returns estimated Groq latency (io + compute - not including runtime) in ms
+    """
+    try:
+        gmodel = groqmodel.load(model_folder, cache_folder)
+        if (
+            gmodel.state.info.assembler_success  # pylint: disable=singleton-comparison
+            == True
+        ):
+            return 1000 * gmodel.estimate_performance().latency
+        else:
+            return "-"
+    except FileNotFoundError:
+        return "-"
 
 
-def _cleanOpList(input):
-    if input is None:
-        return "3P_failure"
-    elif input == []:
-        return "-"
-    else:
-        return "; ".join(map(str, input))
+def get_report_name() -> str:
```

Comment:
```suggestion
def _get_report_name() -> str:
```

Reply:
This is actually used by test/cli as well to check for the generated report file.